lua/tree.lua

```diff
@@ -55,6 +71,10 @@ function M.on_keypress(mode)
     return lib.toggle_ignored()
   end
 
+  if mode == 'next_git_item' or mode == 'prev_git_item' then
+    return go_to_git_item(mode)
+  end
```

Comment:
and then you could load specific function if it's next or prev.

Reply:
I'm using separate if blocks now to call cached functions.